dev-utils/src/commands/init_command.rs

```diff
@@ -199,21 +197,6 @@ impl InitCommand {
             "these incentive wallets are special, cannot be withdrawn (anykey)",
```

Comment:
this should be also removed -- did you try running the dev-utils commands?

Reply:
no, I'll fix it.